data/json/npcs/TALK_REFUGEE_BEGGAR_1.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_BEGGAR_1",
+    "dynamic_line": {
+      "u_is_wearing": "badge_marshal",
```

Comment:
Reasonable, although I think I'll leave it for now, as this is consistent with the refugee center dialogue overall. I'll probably fix it all in one big PR for everyone so that it can be consistent.